src/ids-modal/ids-modal-z-counter.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Provides a global way to display
+ */
+class IdsModalZCounter {
+  constructor() {
+    this.zCounter = 1000;
```

Comment:
```suggestion
    this.zCounter = 1020;
```

Minor but the z-indexes i worked out start modals at 1020